package.json

```diff
@@ -63,7 +62,7 @@
     "tests:integration:debug": "DEBUG=true ./test/dockerized-integration-tests.sh",
     "compile": "webpack",
     "watch": "webpack --watch",
-    "postinstall": "cd modules/electrophysiology_browser/jsx/react-series-data-viewer && npm ci"
+    "postinstall": "cd modules/electrophysiology_browser/jsx/react-series-data-viewer && npm install"
```

Comment:
Does this to be changed given that you commited the new package-lock.json? I'm afraid that it might get accidentally pushed forward in a branch merge.